### ISSUE TITLE
turf.feature not defined - (doc outdated ?)

### ISSUE THREAD

**Issue by Tazaf** [Turfjs/turf#406]
Hi !

The [documentation](http://turfjs.org/docs/#feature) describes a helper function named `feature` that transforms a geometry into a feature GeoJSON object.

Except that this function seems to be undefined.

I tried this code, same as in the documentation, with the minified version of turf (downloaded from https://wzrd.in/standalone/turf@latest, as suggested by [orangemug](https://github.com/orangemug) in [this issue](https://github.com/Turfjs/turf/issues/379)):

```javascript
var geometry = {
     "type": "Point",
     "coordinates": [
       67.5,
       32.84267363195431
     ]
   }

var feature = turf.feature(geometry);
```

But ended up with this error:
```
TypeError: turf.feature is not a function
```

Maybe this `feature` function has been removed, since every other helper function returns a Feature ? If this is the case, the documentation could be updated to reflect this change.
If this is not the case, maybe I missed something ?

**Comment by tcql** [comment]
Looks like this is fixed in #394, but a new release hasn't been rolled. If you use the `turf-helpers` directly (https://wzrd.in/standalone/turf-helpers@latest looks fine) you can use all of the helper methods right away

**Comment by Tazaf** [comment]
So this will be OK with the next release ? Nice :wink:
In the meantine, using specific helper function will do just fine.
Thanks ! (and closing).
